src/ontology/mondo-ingest.Makefile

```diff
@@ -199,13 +199,16 @@ $(REPORTDIR)/%_mapping_status.tsv $(REPORTDIR)/%_unmapped_terms.tsv: $(REPORTDIR
 	--outpath-simple $(REPORTDIR)/$*_unmapped_terms.tsv \
 	--outpath-full $(REPORTDIR)/$*_mapping_status.tsv
 
-components/%-unmapped.owl: components/%.owl reports/%_unmapped_terms.tsv
+unmapped/:
+	mkdir -p $@
+
+unmapped/%-unmapped.owl: $(COMPONENTSDIR)/%.owl reports/%_unmapped_terms.tsv | unmapped/
```

Comment:
@sabrinatoro We had an issue where an ODK change made it so that the goal to create the `components/%-unmapped.owl` files wasn't working. So we changed the location to a new folder!: `src/ontology/unmapped/`